README.md

```
@@ -4,103 +4,83 @@ This software is an SMS Bridge Consolidator that works on a laptop connected to
-# Step 1: Install K3s (one-time, requires sudo, run from project root, verbose output)
-ansible-playbook -i ansible-k3s/inventory.txt ansible-k3s/install_k3s.yml --ask-become-pass -vvv
-
-# Step 2: Deploy SMS Bridge (vault password only, run from project root, verbose output)
-ansible-playbook -i ansible-k3s/inventory.txt ansible-k3s/setup_sms_bridge_k3s.yml --ask-vault-pass --ask-become-pass -vvv
-```
-
-## Setup Steps
-
-1. **Create Vault File**: Copy and customize the vault file
+1. **Set up configuration**: Copy and customize configuration files
    ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)